Holovibes/sources/thread/information_worker.cc

```diff
@@ -204,16 +204,26 @@ int get_gpu_load(nvmlUtilization_t* gpuLoad)
     return nvmlShutdown();
 }
 
-const std::string get_load_color(float load, float max_load)
+const std::string get_load_color_custom(float load, float max_load, float orange_ratio, float red_ratio)
```

Comment:
You can set default values for `orange_ratio` and `red_ratio` instead of creating a new function `get_load_color`